doc/source/cheat_sheet/cheat_sheet.qmd

```diff
@@ -0,0 +1,275 @@
+#| eval: false
+mapdl.solu() # SOLU
+mapdl.slashsolu() # /SOLU
+
+mapdl.vget() # VGET
+mapdl.starvget() # *VGET
+``` 
+
+Load a table from Python to MAPDL:
+```{python}
+#| echo: false
+mapdl = launch_mapdl()
+```
+```{python}
+import numpy as np
+np_array = np.array([[1,2,3], [4,5,6]])
+mapdl.load_table("table_name", np_array)
+```
+
+Write parameters and access from or  to the MAPDL database:
+```{python}
+# Create a parameter from a NumPy array
+mapdl.parameters['my_np_param'] = np_array
+# Save a parameter to a NumPy array
+saved_np_array = mapdl.parameters['my_np_param']
+```
+
+```{python}
+#| echo: false
+mapdl.exit()
+```
+
+# Convert APDL script to Python files
+
+An existing APDL script can be converted to PyMAPDL format
+with the following commands:
+```{python}
+#| eval: false
+import ansys.mapdl.core as pymapdl
+pymapdl.convert_script(
+  "mapdl_script.dat", "pymapdl_script.py"
+)
+```
+
+It can also be converted using PyMAPDL CLI:
+```{python}
+#| eval: false
+pymapdl convert mapdl_script.dat 
+-o pymapdl_script.py
+```
+
+# Mesh a model
+
+Store the finite element mesh as a 
+[VTK UnstructuredGrid data](https://vtk.org/doc/nightly/html/classvtkUnstructuredGrid.html)
+object:
+
+```{python}
+#| echo: false
+#####################################################################
+# Hidden example for testing purpose (3d_plate_thermal.py)
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+mapdl = launch_mapdl()
+
+# Create a simple beam, specify the material properties, and mesh it.
+mapdl.prep7()
+mapdl.mp("kxx", 1, 45)
+mapdl.et(1, 90)
+mapdl.block(-0.3, 0.3, -0.46, 1.34, -0.2, -0.2 + 0.02)
+mapdl.vsweep(1)
+
+# Set the thermal boundary conditions
+mapdl.asel("S", vmin=3)
+mapdl.nsla()
+mapdl.d("all", "temp", 5)
+mapdl.asel("S", vmin=4)
+mapdl.nsla()
+mapdl.d("all", "temp", 100)
+out = mapdl.allsel()
+
+mapdl.vsweep(1)
+```
+
+```{python}
+grid = mapdl.mesh.grid
+```
+
+Save element and node numbers to Python arrays:
+
+```{python}
+# Get an array of the nodal coordinates
+nodes = mapdl.mesh.nodes
+# Save node numbers of selected nodes to an array
+node_num = mapdl.mesh.nnum
+# Save node numbers of all nodes to an array
+node_num_all = mapdl.mesh.nnum_all
+# Get element numbers of currently selected elements
+elem_num = mapdl.mesh.enum
+# Get all element numbers, including those not selected
+elem_num_all = mapdl.mesh.enum_all
+```
+
+# Solve an analysis
+
+```{python}
+mapdl.slashsolu()
+mapdl.solve()
+mapdl.finish()
+```
+
+# Post-process results
+
+The
+[PostProcessing](https://mapdl.docs.pyansys.com/version/stable/api/_autosummary/ansys.mapdl.core.post.PostProcessing.html#ansys.mapdl.core.post.PostProcessing)
+class is used for plotting and saving results to NumPy arrays.
+
+```{python}
+mapdl.post1()
+mapdl.set(1, 1)
+mapdl.allsel()
+
+# Plot nodal temperatures
+mapdl.post_processing.plot_nodal_temperature()
+# Save nodal temperatures to a Python array
+nodal_temp= mapdl.post_processing.nodal_temperature()
```

Comment:
Can you add also something about `to_list` or `to_array` ?? https://mapdl.docs.pyansys.com/version/stable/user_guide/post.html#enriched-command-output